modules/core/server/views/layout.server.view.html

```diff
@@ -1,5 +1,5 @@
 ﻿<!DOCTYPE html>
-<html lang="en">
+<html lang="en" {% if env === "production" %}ng-strict-di{% endif %}>
```

Comment:
We could just annotate in development too, which I'm not opposed to. Honestly I think ngStrictDi would prevent errors in production if we also check it in development.